server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts

```diff
@@ -241,7 +241,8 @@ export class TransformHandler {
 
     async pollTransformation(request: GetTransformRequest, validExitStatus: string[], failureStates: string[]) {
         let timer = 0
-
+        let getTransformRetry = 0
```

Comment:
thanks for adding a logic here to have retries. also can we also explore if this will be helpful if we add similar retries in getTransformPlan and CancelTransform

Reply:
If we add this logic to other functions, consider creating a wrapper function with retries that takes any function as a parameter along with the number of retries.